src/rust/engine/engine_cffi/src/lib.rs

```diff
@@ -601,17 +606,30 @@ fn workunits_to_py_tuple_value<'a>(workunits: impl Iterator<Item = &'a Workunit>
 pub extern "C" fn poll_session_workunits(
   scheduler_ptr: *mut Scheduler,
   session_ptr: *mut Session,
+  max_log_verbosity_level: u64,
 ) -> Handle {
+  let py_level: Result<PythonLogLevel, _> = max_log_verbosity_level.try_into();
+  let max_log_verbosity: log::Level = match py_level {
+    Ok(level) => level.into(),
+    Err(e) => {
+      println!(
```

Comment:
Should avoid println in favor of logging: `warn!` probably.

Better yet would be to actually error here, but not a blocker.